website/source/docs/providers/chef/r/data_bag.html.markdown

```diff
@@ -6,7 +6,7 @@ description: |-
   Creates and manages a data bag in Chef Server.
 ---
 
-# chef\_data\_bag
+# chef_data_bag
```

Comment:
hmm we used to do this because markdown would interpret the underscores as emphasis. Is that no longer true?